builder/dockerfile/buildargs.go

```diff
@@ -17,6 +17,8 @@ var builtinAllowedBuildArgs = map[string]bool{
 	"https_proxy": true,
 	"FTP_PROXY":   true,
 	"ftp_proxy":   true,
+	"ALL_PROXY":   true,
+	"all_proxy":   true,
```

Comment:
And SOCKS_PROXY and RSYNC_PROXY?

Reply:
Didn't know about those.. hmf.. not happy there's so many